test/e2e/integration/commands/init.test.ts

```diff
@@ -23,11 +23,13 @@ import {
   DependencyManager
 } from '../../../../src/core/dependency_managers/index.js'
 import {
-  ChartManager, ConfigManager, constants, Helm, K8, KeyManager, LeaseManager, logging, PackageDownloader, Zippy
+  ChartManager, ConfigManager, constants, Helm, K8, KeyManager, LeaseManager, logging, PackageDownloader, Zippy, LocalConfig,
 } from '../../../../src/core/index.js'
 import { SECONDS } from '../../../../src/core/constants.js'
 import sinon from 'sinon'
 import { IntervalLeaseRenewalService } from '../../../../src/core/lease/lease_renewal.js'
+import path from "path";
```

Comment:
you are getting code style standard errors.  I think if you run `npm run format` it might fix a lot of these.